packages/runtime/src/binding/computed-observer.ts

```diff
@@ -22,7 +22,7 @@ export function computed(config: ComputedOverrides): PropertyDecorator {
   };
 }
 
-const noProxy = !(typeof Proxy !== undefined);
+const noProxy = !(Proxy !== undefined);
```

Comment:
```suggestion
const noProxy = !(typeof Proxy !== 'undefined');
```
This will throw a reference error if `Proxy` really isn't defined otherwise

Reply:
Using `typeof` triggers another linting error so thats why I solved it this way, but this is a valid reason for suppressing.

Just pushed the change.